backend/app/Repositories/EloquentSessionRepository.php

```diff
@@ -6,11 +6,27 @@
 
 use App\Repositories\Contracts\SessionRepository;
 use Illuminate\Support\Collection;
+use Illuminate\Support\Carbon;
+use App\Entities\Session;
+use App\Actions\Sessions\GetAvgSessionRequest;
+use App\Entities\Visitor;
+use Illuminate\Support\Facades\DB;
+use App\Actions\Sessions\AverageSessionFilter;
 
 class EloquentSessionRepository implements SessionRepository
 {
     public function getCollection(): Collection
     {
         return collect([]);
     }
+
+    public function getAvgSession(AverageSessionFilter $filter): Collection
+    {
+        return DB::table('sessions')
+                ->selectRaw('EXTRACT(EPOCH FROM (AVG(end_session - start_session))) as avg')
```

Comment:
I cannot understand, if you're counting average here, what are you doing in the response then?

Reply:
Here i get integer, for ex. 100 sec. You asked response has to be in timestamp UTC, therefore I convert seconds to timestamp in the response and get: ['avg_sessions => '1970-01-01 00:01:40'']